### PR TITLE
Start testing with GCC 11 and a small pkgconfig fix

### PR DESCRIPTION
**Pull request by tuliom**
1. Start running the tests on GCC 11.
2. Guarantee the pkgconfig directory is created before we try to install files in it.

### CODE REVIEW

**Comment by tuliom** [comment]
There is an issue with the Travis CI settings for this repository.
I've tested this PR in my own repo and it works: https://travis-ci.com/github/tuliom/libdfp/builds/229951284

File: Makefile.in [hunk 1]
```diff
@@ -446,6 +446,7 @@ endif
 	$(INSTALL_DATA) -t $(inst_libdir) $(top_builddir)/libdecnumber/libdecnumber.a
 	ln -f -s $(SHARED_REALNAME_LIB) $(inst_libdir)/$(SHARED_LINKERNAME_LIB)
 	ln -f -s $(SHARED_REALNAME_LIB) $(inst_libdir)/$(SHARED_SONAME_LIB)
+	$(INSTALL) -d $(inst_libdir)/pkgconfig
 	$(INSTALL_DATA) -D -t $(inst_libdir)/pkgconfig $(top_builddir)/libdfp.pc
```
Comment (pmur) [inline Makefile.in:450]: Huh, that seems weird. Is there some racy condition between the two install commands below? I guess we don't need the -D options on the pkgconfig files below if we add this.

Comment (tuliom) [reply Makefile.in:450]: Indeed. It does look something else caused the issue.
Let me remove this patch, leave just the GCC 11 test one and try to fix the CI integration.

**Review by pmur — APPROVED** [review]

**Comment by tuliom** [comment]
All tests passed!
Merging...

**Pull request merged**